src/loader/tsbgw_interface.h

```diff
@@ -0,0 +1,22 @@
+#ifndef TSBGW_INTERFACE_H
+#define TSBGW_INTERFACE_H
+
+#include <postgres.h>
+#include <fmgr.h>
+
+/* This is where versioned-extension facing functions live. It shouldn't live anywhere else */
+
+PG_FUNCTION_INFO_V1(tsbgw_worker_reserve);
```

Comment:
I was under the impression I couldn't use that in the loader...

Reply:
You should be able to since it is just a macro you can include. But maybe it isn't needed here? I figured our macro had the benefit of ensuring some extra symbol export required on Windows.